lib/cc/analyzer/engine.rb

```diff
@@ -38,7 +38,7 @@ def run(stdout_io, container_listener)
           output = EngineOutput.new(raw_output)
 
           unless output.valid?
-            stdout_io.failed("#{qualified_name} produced invalid output: #{output.error[:message]}")
+            stdout_io.errored("#{qualified_name} produced invalid output: #{output.error[:message]}")
```

Comment:
I'm :+1: on this cleanup. Are there builder-side changes to conform to the new Formatter interface?